src/modules/addTodo.js

```diff
@@ -0,0 +1,31 @@
+import TodoList from './todoList.js';
+import getTodo from './getTodo.js';
+import setTodo from './setTodo.js';
+import renderAddedList from './renderList.js';
+
+// This code  defines an addToBtn function that is used to add a new todo item to a todo list.
+// When the function is called,it adds an event listener to the input element with the class input.
+// When the user presses the Enter key while the input element is focused, the event listener is
+// triggered.
+const addToBtn = () => {
+  const input = document.querySelector('.save-todo');
+  input.addEventListener('keypress', (event) => {
+    if (event.key === 'Enter') {
+      event.preventDefault();
+      const description = document.querySelector('.save-todo').value;
+      const completed = false;
+      const list = getTodo();
+      const index = list.todoSize + 1;
+      if (description !== '') {
+        const todoList = new TodoList(description, completed, index);
```

Comment:
* Kindly fix this bug 🐛  I get when I try to add more todos to the todo list below is a recording of the said bug 👇 

https://user-images.githubusercontent.com/52497246/208034431-0ea5eb77-f492-4de2-a252-09c715f88bbc.mp4